RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate SettingsView to TypeScript

## Proposed changes


## Issue(s)	


## How to test or reproduce

On Settings:

1. Check that each button works properly
2. Check that `log out` and `clean cache` are working properly

## Screenshots

## Types of changes




## Checklist



## Further comments

Needs `utils` to be migrated to TypeScript
